### DISCUSSION TITLE
Orcid id and citation as in a distill article in quarto blog posts

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#631] [Q&A] [answered]
Is it possible to include the orcid id and the cite of the post at the end of the article as in a distill article?

**Comment by dragonstyle** [comment, marked as answer]
Yes you can! Here is more about creating the cite in the appendix:

https://quarto.org/docs/authoring/create-citeable-articles.html

For orcids, you can include them with authors like:

```
author:
  - name: Norah Jones
    orcid: <orcid>
```

And the should be emitted. If you include a DOI in the header like:

```
title: foo
doi: 10/120.2323.242323
```

That should also be rendered appropriately.

HTH!

**Comment by ManuelSpinola** [comment]
Thank you very much.
